cli/configssh.go

```diff
@@ -501,6 +508,14 @@ func (r *RootCmd) configSSH() *clibase.Cmd {
 			Description: "Specifies the path to an SSH config.",
 			Value:       clibase.StringOf(&sshConfigFile),
 		},
+		{
+			Flag:    "coder-cli-path",
+			Env:     "CODER_SSH_CONFIG_CLI_PATH",
+			Default: "",
+			Description: "Optional to specify the path for the coder cli uses in ProxyCommand. " +
+				"By default, the coder cli used is the same cli being invoked with 'config ssh'.",
```

Comment:
If we change the cli->binary terminology:

```suggestion
			Description: "Optionally specify the path to the coder binary used in ProxyCommand. " +
				"By default, the binary invoking this command ('config ssh') is used.",
```